setup.py

```diff
@@ -54,12 +54,12 @@ def desc():
         "Flask-Login>=0.3, <0.5",
         "Flask-OpenID>=1.2.5, <2",
         "Flask-SQLAlchemy>=2.4, <3",
-        "Flask-WTF>=0.14.2, <1",
+        "Flask-WTF>=0.14.2, <0.15.0",
         "Flask-JWT-Extended>=3.18, <4",
         "jsonschema>=3.0.1, <4",
         "marshmallow>=3, <4",
         "marshmallow-enum>=1.5.1, <2",
-        "marshmallow-sqlalchemy>=0.22.0, <1",
+        "marshmallow-sqlalchemy>=0.22.0, <0.24.0",
```

Comment:
@dpgaspar Why is 0.24.0 not allowed?

0.24.0 only [drops support for marshmallow 2](https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/dev/CHANGELOG.rst#0240-2020-10-20), but FAB doesn't support ma 2 anyway.

Reply:
The reasoning was that 0.23.X was tested but not 0.24